test.js

```diff
@@ -0,0 +1,7 @@
+function hello(name) {
+    document.body.textContent = "Hello, " + nama + "!"
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
'nama' is not defined             no-undef